app/controllers/userController.ts

```diff
@@ -0,0 +1,66 @@
+import { UserService } from "../services/UserService";
+import User from "../models/user";
+import mongoose from "mongoose";
+import { AppResponse } from "../services/app-responses";
+
+export class UserController {
+  public userService: UserService = new UserService();
+  public response: AppResponse = new AppResponse();
+
+  public getAllUsers = async (req, res, next) => {
+    try {
+      const getAllUsers = await this.userService.getAllUsers();
+      return this.response.successOk(res, "", getAllUsers);
+    } catch (error) {
+      return this.response.errorOnServer(res, "Internal Server Error", error);
+    }
+  };
+
+  public createUser = async (req, res, next) => {
+    try {
+      const userObject = new User({
+        _id: new mongoose.Types.ObjectId(),
+        name: req.body.name,
+        age: req.body.age,
+      });
+      const createdUser = await this.userService.createNewUser(userObject);
+      return this.response.successOnCreate(
+        res,
+        "New User Created Succesfully",
+        createdUser
+      );
+    } catch (error) {
+      return this.response.errorOnServer(res, "Internal Server Error", error);
+    }
+  };
+
+  public getUserById = async (req, res, next) => {
```

Comment:
@Suresh-ashwini Have type in argument ...Types are main reason for using typescript

Reply:
Updated the argument type for parameters